starter_code/hangman.js

```diff
@@ -1,24 +1,36 @@
 var hangman;
 
-// function Hangman() {
+function Hangman() {
+ this.words = ['IRONHACK', 'NODEJS', 'JAVASCRIPT', 'METEOR', 'ANGULAR', 'BARCELONA', 'MADRID', 'MIAMI', 'HTML'];
+ this.secretWord = '';
+ this.letters = [];
+ this.guessedLetter = '';
+ this.errorsLeft = 10;
+//  this.secretWord =this.getWord();
 
-// }
+}
 
-// Hangman.prototype.getWord = function () {
+Hangman.prototype.getWord = function () {
+  var randomNumber = Math.floor(Math.random()* this.words.length);
 
-// };
-
-// Hangman.prototype.checkIfLetter = function (keyCode) {
-
-// };
-
-// Hangman.prototype.checkClickedLetters = function (key) {
+  return this.words[randomNumber];
+};
 
-// };
+Hangman.prototype.checkIfLetter = function (keyCode) {
+    return keyCode > 64 && keyCode < 91;
+};
 
-// Hangman.prototype.addCorrectLetter = function (i) {
+Hangman.prototype.checkClickedLetters = function (key) {
+    if (this.letters.includes(key)) {
+      return false;
+    } else {
+      return true;
+    }
+};
 
-// };
+Hangman.prototype.addCorrectLetter = function (i) {
+  this.guessedLetter = this.guessedLetter + i;
```

Comment:
Here "they" pass you `i` which is the  index of the letter in the `secretWord` variable. So you should add that letter in the `guessedLetter`